git-rebase--interactive.sh

```diff
@@ -87,16 +70,16 @@ git_rebase__interactive () {
 	git rebase--helper --prepare-branch "$switch_to" ${verbose:+--verbose}
 	init_basic_state
 
-	init_revisions_and_shortrevisions
-
 	git rebase--helper --make-script ${keep_empty:+--keep-empty} \
 		${rebase_merges:+--rebase-merges} \
 		${rebase_cousins:+--rebase-cousins} \
-		$revisions ${restrict_revision+^$restrict_revision} >"$todo" ||
+		${upstream:+--upstream "$upstream"} ${onto:+--onto "$onto"} \
+		${squash_onto:+--squash-onto "$squash_onto"} \
+		${restrict_revision+^$restrict_revision} >"$todo" ||
```

Comment:
Oh, so that is the piece that I was missing. You changed the signature of the `--make-script` call!

Maybe it would make sense to split this commit into two, even, to first implement the part that is needed by `--make-script`, and the second commit that takes care of the needs of `--complete-action` (most notably, the `shortrevisions` part).

Would make it easier to review, methinks.